### ISSUE TITLE
-G mode issue MacOS

### ISSUE THREAD

**Issue by nupator** [Ettercap/ettercap#1041]
hello
my etternet has an issue
im not a pr0 programmer just a beginner. im 16. dont expect much :]
so i run it using this command in terminal 👨‍💻 

```
sudo ettercap
sudo etternet
```

Recreating my issue:

(1)
<img width="36" alt="Screen Shot 2020-05-27 at 8 01 37 PM" src="https://user-images.githubusercontent.com/40234755/83044315-113b8880-a055-11ea-9cae-6545508210f9.png">

(2)
<img width="175" alt="Screen Shot 2020-05-27 at 8 01 43 PM" src="https://user-images.githubusercontent.com/40234755/83044697-9888fc00-a055-11ea-9f75-57a30e41f792.png">

(3)
<img width="157" alt="Screen Shot 2020-05-27 at 8 01 49 PM" src="https://user-images.githubusercontent.com/40234755/83044772-b22a4380-a055-11ea-82b6-02af233ce814.png">

(4)
<img width="434" alt="Screen Shot 2020-05-27 at 8 02 26 PM" src="https://user-images.githubusercontent.com/40234755/83044896-dd149780-a055-11ea-884d-24ed4da9b57b.png">

what should i do?

anything would be helpful :]

(maybe someone addressed this issue before but the soluiton was probably for linux. thanks in advance)

**Comment by koeppea** [comment]
Do you want to open a custom plugin or start a plugin that comes along with ettercap?
Try menubutton -> Plugins -> Plugin list. Then double click the plugin you want to start.
The "Load Plugin" option is only required if you have written and compiled your own plugin and want to load it.

**Comment by koeppea** [comment]
Sorry it's called "Manage plugins"

**Comment by nupator** [comment]
@koeppea 
y cant i load plugins i downloaded thru the internet tho. its like the program doesnt have full disk access

**Comment by koeppea** [comment]
That’s right. It’s dropping the rights to nobody. You can try to set the UID to the UID of your user.
